src/SpeechHoya.php

```diff
@@ -0,0 +1,27 @@
+<?php
+namespace OhSky\DocomoApi;
+
+/**
+ * @see https://dev.smt.docomo.ne.jp/?p=docs.api.page&api_name=text_to_speech&p_name=api_hoya#tag01
+ */
+class Talk extends DocomoApi {
```

Comment:
__[code_sniffer]__
* Opening brace of a class must be on the line after the definition